Donation-platform/src/main/java/com/google/DbSetUpUtils.java

```diff
@@ -69,12 +74,12 @@ private void addTags() {
   private void addCharities() {
     try {
       // translate tags to keys
-      Key hunger = db.getTagByName("hunger").getId();
-      Key education = db.getTagByName("education").getId();
-      Key children = db.getTagByName("children").getId();
-      Key environment = db.getTagByName("environment").getId();
-      Key racialEquality = db.getTagByName("racial equality").getId();
-      Key health = db.getTagByName("health").getId();
+      Tag hunger = db.getTagByName("hunger");
```

Comment:
I see Ryan is also making the same change in his PR. So one of you will get merge conflict. But that's ok, you can resolve it.

Reply:
yes, we'll resolve when we get there